lib/validate_partitions.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package validate_partitions;
+
+use base "opensusebasetest";
+
+use strict;
+use warnings;
+use scheduler 'get_test_suite_data';
+use testapi;
+
+my $test_data = get_test_suite_data();
+
+# If we want to validate Msdos partition
+sub validate_partition_table {
+    my $root_disk = script_output("mount |grep \" / \"");
```

Comment:
you don't need to escape anything here: `'mount | grep /'`

Reply:
Done